product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/pgbouncer_configuring.mdx

```diff
@@ -5,6 +5,8 @@ navTitle: Configuring EDB PgBouncer
 
 [EDB PgBouncer](/pgbouncer/latest/) is a lightweight connection pooling utility that you can use to more quickly connect to your BigAnimal cluster. It saves you time by pooling previously established connections to your server. Instead of connecting directly to your server, EDB PgBouncer negotiates your connections for you. 
 
+Additionally, you can enable PgBouncer and configure your settings in [Creating a cluster](../../getting_started/creating_a_cluster/pgbouncer).
```

Comment:
I think we want to explain about how these instructions are for the client and the instructions on the Create a cluster topic is for the server.